torch_geometric/sampler/utils.py

```diff
@@ -68,7 +68,12 @@ def to_csc(
         row, col = data.edge_index
         if not is_sorted:
             row, col, perm = sort_csc(row, col, src_node_time)
-        colptr = torch.ops.torch_sparse.ind2ptr(col, data.size(1))
+
+        if hasattr(torch, '_convert_indices_from_coo_to_csr'):
+            colptr = torch._convert_indices_from_coo_to_csr(
```

Comment:
Curious is it supported in torch with later versions so we have that if-else?

Reply:
Yes, this is needed for PyTorch 1.10 support :D